src/main/scala/com/elarib/PartialSbtPlugin.scala

```diff
@@ -144,13 +149,19 @@ object PartialSbtPlugin extends AutoPlugin {
 
         val diffsFiles: Seq[sbt.File] = changeGetter.changes.filterNot(f => isFileExcluded(baseDir)(f, excludedFiles))
 
+        def isExcludedTestFile(file: sbt.File, resolvedProject: ResolvedProject): Boolean =
+          excludeTestFiles && file.getAbsolutePath.startsWith((resolvedProject.base / "src" / "test").getAbsolutePath)
```

Comment:
Let's retrieve all the test directories from SBT (there could be more than one test directory, for example, java sources) — `Test / sourceDirectory`. WDYT?

Reply:
I was thinking about it, but there's an issue.
In order to access `Test / sourceDirectories` of a certain `ResolvedProject` we need an instance of `Settings[Scope]`, which is provided by an instance of `BuildStructure`, which we can get by evaluating the `buildStructure` **task**. 
However `changedProjects` is a command and it's defined by appending an instance of `Command` to the `commands` **setting**. 
The problem is that a setting could not depend on a task and rewriting `changedProjects` to a task is way outside of the scope of this PR.

BTW `src/test` covers all **default** test directories including `java` and `scala-*`. Which is our case and let it be a limitation of this plugin :)